kythe/go/extractors/config/smoke/smoke.go

```diff
@@ -38,46 +38,77 @@ import (
 // automatically. The current implementation is to simply check the fraction of
 // files covered by extractor output.
 //
-// TODO(danielmoy): hook up an indexing step and check actual semantic output
-// instead of simple files.
+// Also optionally supports indexing the extractors output.
 //
 // TODO(danielmoy): support more than just java.
```

Comment:
I think this comment is no longer needed?

Reply:
ah I think that's still needed

Reply:
Maybe—but maybe somewhere else? It's not really part of the `Tester` documentation.